core/src/main/java/org/apache/calcite/adapter/enumerable/EnumUtils.java

```diff
@@ -786,28 +787,29 @@ static Expression tumblingWindowSelector(
     final Expression wmColExprToLong = EnumUtils.convert(wmColExpr, long.class);
     final Expression shiftExpr = Expressions.constant(1, long.class);
 
-    // Find the fixed window for a timestamp given a window size and return the window start.
-    // Fixed windows counts from timestamp 0.
-    // wmMillis / intervalMillis * intervalMillis
-    expressions.add(
-        Expressions.multiply(
-            Expressions.divide(
+    // Find the fixed window for a timestamp given a window size and an offset and return the
```

Comment:
"...a window size and an offset and return..." -> "a window size and an offset, and return"

Reply:
Done.